infra.bs

```diff
@@ -25,6 +25,8 @@ Boilerplate: omit conformance, omit feedback-header, omit idl-index
 
 <pre class="anchors">
 urlPrefix: https://tc39.github.io/ecma262/; spec: ECMA-262; type: dfn
+    text: %JSONParse%; url: sec-json.parse
+    text: Call; url: sec-call
```

Comment:
`Call` should be `abstract-op`, not `dfn`

I guess we don't have a good name for stand-alone functions, so we can leave that as a dfn, despite that causing casing issues in "terms referenced by this specification". /cc @tabatkins